internal/uvm/create_lcow.go

```diff
@@ -738,6 +719,12 @@ func makeLCOWDoc(ctx context.Context, opts *OptionsLCOW, uvm *UtilityVM) (_ *hcs
 			}
 		} else {
 			kernelArgs = "root=/dev/sda ro rootwait init=/init"
+			if opts.DmVerityMode {
+				if len(opts.DmVerityCreateArgs) == 0 {
+					return nil, errors.New("DmVerityCreateArgs must be set when DmVerityMode is true")
```

Comment:
"DmVerityCreateArgs must be set when DmVerityMode is true and not booting from a vmgs file.'